src/main/java/seedu/bankwithus/BankWithUs.java

```diff
@@ -36,9 +35,25 @@ public BankWithUs(String filePath) throws IOException {
         }
     }
 
+    /**
+     * exit the programme, save the data and show farewell message
```

Comment:
Just me being really really anal, but yea, exit should be Exits. Obviously no need to change this